### ISSUE TITLE
Flutter Doctor -v Android Studio: Unable to find bundled Java version

### ISSUE THREAD

**Issue by gab-luz** [flutter/flutter#113161]
## Steps to Reproduce

1. open up a terminal
2. type flutter doctor -v

## Logs

<details>

<summary>Logs</summary>

```


```

```
flutter doctor --verbose
[✓] Flutter (Channel stable, 3.3.4, on Linux Mint 21 5.15.0-48-generic, locale pt_BR.UTF-8)
    • Flutter version 3.3.4 on channel stable at /home/mestre/.asdf/installs/flutter/3.3.4-stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision eb6d86ee27 (3 days ago), 2022-10-04 22:31:45 -0700
    • Engine revision c08d7d5efc
    • Dart version 2.18.2
    • DevTools version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at /home/mestre/Android/Sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: /usr/lib/jvm/java-1.8.0-openjdk-amd64/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_342-8u342-b07-0ubuntu1~22.04-b07)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /home/mestre/Downloads/AppImages/Google-Chrome-Beta.AppImage

[✓] Linux toolchain - develop for Linux desktop
    • Ubuntu clang version 14.0.0-1ubuntu1
    • cmake version 3.24.2
    • ninja version 1.10.1
    • pkg-config version 0.29.2

[!] Android Studio (version 2022.2)
    • Android Studio at /opt/android-studio-2022.2.1/android-studio
    • Flutter plugin version 70.2.5
    • Dart plugin version 222.4167.21
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[!] Android Studio
    • Android Studio at /opt/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Linux Mint 21 5.15.0-48-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 107.0.5304.18 beta

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 2 categories.
```

</details>
I can't make flutter doctor work without issues. When I run flutter doctor -v, it seems like it can't find JDK installed, but it is installed.
My .bashrc has:
export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
export PATH=$PATH:$JAVA_HOME/bin

**Comment by exaby73** [comment]
Hello @gab-luz. From your doctor logs, I can't see any issues. can you share the error doctor logs?

**Comment by exaby73** [comment]
Yes that's the right command but the logs you included in https://github.com/flutter/flutter/issues/113161#issue-1401924376 doesn't show any errors in the command.

**Comment by gab-luz** [comment]
> Yes that's the right command but the logs you included in [#113161 (comment)](https://github.com/flutter/flutter/issues/113161#issue-1401924376) doesn't show any errors in the command.

It's not finding my java install.

**Comment by gab-luz** [comment]
I'll close it for now. Thanks bro.
